build-frontend/action.yml

```diff
@@ -34,45 +31,32 @@ inputs:
     description: 'NPM registry'
     required: false
     default: 'https://registrynpm.storefrontcloud.io'
-  api_base_url:
-    description: 'Base Middleware API URL for Storefront. By default https://project_name.region.gcp.storefrontcloud.io/api'
-    required: false
-  multistore_enabled:
```

Comment:
out of curiosity, how those env will be passed then into docker containers?

Reply:
All will be set in Console ;)